.ci/jenkins/config/branch.yaml

```diff
@@ -1,78 +1,110 @@
+    env_vars:
+      NATIVE: true
+      BUILD_MVN_OPTS: -Dproductized -Ddata-index-ephemeral.image=quay.io/kiegroup/kogito-data-index-ephemeral
+      BUILD_MVN_OPTS_CURRENT: -Dnative -Dquarkus.native.container-build=true
+      QUARKUS_BRANCH: '2.13'
+      QUARKUS_NATIVE_BUILDER_IMAGE: mandrel
+      ADDITIONAL_TIMEOUT: 720
+    ids:
+    - native
+    - prod
+    - lts
+  kogito-bdd:
+    auto_generation: false
+    ids:
+    - cloud
+  ecosystem:
+    auto_generation: false
+    ids:
+    - ecosystem
+
+productized_branch: false
 disable:
   triggers: false
 repositories:
 - name: optaplanner
-  branch: main
 - name: optaweb-vehicle-routing
-  branch: main
 - name: optaplanner-quickstarts
-  branch: development
 - name: optaplanner-website
-  branch: main
 git:
   author:
-    name: kiegroup
-    credentials_id: kie-ci
-    token_credentials_id: kie-ci3-token
-  bot_author:
-    name: bsig-gh-bot
-    credentials_id: bsig-gh-bot
-  fork_author:
-    name: kie-ci
-    credentials_id: kie-ci
-  quarkus:
+    name: radtriste
+    credentials_id: radtriste
+    token_credentials_id: radtriste-gh-token
+  bot_author: # This is meant to disappear in the near future
+    name: radtriste-bot-account
+    credentials_id: radtriste-bot
+  fork_author: # Used on repository where a fork is needed
+    name: radtriste-bot-account
+    credentials_id: radtriste-bot
+  quarkus: # Used by quarkus platform jobs
     author:
-      name: quarkusio
-      credentials_id: kie-ci
+      name: radtriste
+      credentials_id: radtriste
     branch: main
   jenkins_config_path: .ci/jenkins
 buildchain_config:
   git:
     repository: optaplanner
     file_path: .ci/buildchain-config.yaml
 maven:
-  settings_file_id: kogito_release_settings
+  settings_file_id: 2bd418aa-56fa-4403-9232-8c77a50fc528
   nexus:
-    release_url: https://repository.jboss.org/nexus
+    release_url: https://repository.stage.jboss.org/nexus
     release_repository: jboss-releases-repository
-    staging_profile_url: https://repository.jboss.org/nexus/content/groups/kogito-public/
+    staging_profile_url: https://repository.stage.jboss.org/nexus/content/groups/kogito-public/
     staging_profile_id: 2161b7b8da0080
-    build_promotion_profile_id: ea49ccd6f174
-  artifacts_repository: ''
+    build_promotion_profile_id: 1966c60aff37d
+  artifacts_repository: http://nexus3-tradisso-nexus.apps.kogito-cloud.hosted.psi.rdu2.redhat.com/repository/kogito-test/
+  #artifacts_repository: ''
   pr_checks:
     repository:
       url: https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/repositories/kogito-runtimes-pr-full-testing/
       creds_id: unpacks-zip-on-qa-nexus
 cloud:
   image:
-    registry_credentials: optaplanner-quay-token
+    registry_credentials: tradisso_registry_token
     registry: quay.io
-    namespace: optaplanner
+    namespace: tradisso
     latest_git_branch: main
 jenkins:
-  email_creds_id: OPTAPLANNER_CI_EMAIL
+  email_creds_id: KOGITO_CI_EMAIL_TO_PERSO
```

Comment:
Kogito?

Reply:
test commit. That is why I am still on draft status :)